src/App.jsx

```diff
@@ -30,21 +84,9 @@ export const App = () => (
 
     <table className="table">
       <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+        {goods && goods.map(good => (
```

Comment:
goods it is an array if(array) always will be true, if you want to check if something exist in array use array.length

Reply:
but goods it is static constant this check is redundant